components/aws/sagemaker/tests/integration_tests/utils/__init__.py

```diff
@@ -0,0 +1,119 @@
+import os
+import subprocess
+import pytest
+import tarfile
+import yaml
+
+# https://github.com/aws/sagemaker-python-sdk/blob/fbebd805e6efc212211b87eafbff3dc84c1062b9/src/sagemaker/amazon/amazon_estimator.py
+KMEANS_REGISTRY = {
```

Comment:
you can use `from sagemaker.amazon.amazon_estimator import get_image_uri` and use that instead of maintaining it. You will have to add sagemaker sdk.